packages/cli/src/__tests__/commands/build-docs.test.ts

```diff
@@ -51,7 +51,7 @@ describe('build-docs', () => {
       disableGoogleFont: false,
       template: '',
       templateOptions: {},
-      features: { openapi: {} },
```

Comment:
Now we're clear that it should be:

`theme: { openapi: {} },`

Shall we finish this adjustment?

Reply:
Great! I'll finish that.